_authors/arnavgrover.markdown

```diff
@@ -0,0 +1,8 @@
+---
+name: Arnav Grover
+short_name:
+twitter: 
```

Comment:
Please remove the twitter tag and recommit and we should be good!

Reply:
Just recommitted!